CONTRIBUTING.md

```diff
@@ -91,6 +118,38 @@ Once approved, the author of the pull request,
 or reviewer if the author does not have commit access,
 should "Squash and merge".
 
+### Design considerations
+
+#### Performance
+
+The agent is designed to monitor production applications. Therefore it's very important to keep the overhead of the agent as low as possible.
```

Comment:
"the overhead" -> "the performance overhead"

Reply:
Fixed